### PR TITLE
Add ubuntu k8s french engage page

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add french Ubuntu k8s engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/fr/canonical-ajoute-containerd-a-ubuntu-kubernetes
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the page matches [the copy doc](https://docs.google.com/document/d/12j3_XhZkNwAy7WTgpe9t5THJU84EtaA1VEyQ86VOgOI/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2751

### CODE REVIEW

**Comment by tbille** [comment]
QA-1 because there are a lot of typos/sentences that doens't mean anything. Will go and propose some copy docs propsals

**Comment by steverydz** [comment]
@deadlight this is ready for re-review

File: templates/engage/fr/canonical-ajoute-containerd-a-ubuntu-kubernetes.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          alt="",
+          height="269",
+          width="501",
+          hi_def=True,
+          attrs={"class": ""},
+          loading="lazy",
+        ) | safe
+      }}
+      <p>Canonical fournit maintenant un support pour <a href="https://containerd.io/">containerd</a> dans la version 1.14 de <a href="http://www.ubuntu.com/kubernetes">Charmed Kubernetes</a> et de <a href="http://microk8s.io/">Microk8s</a>, améliorant la sécurité et la stabilité.</p>
+      <p>“ Containerd est devenu le standard de l'’industrie du container runtime. Il est concentré sur la simplicité, la stabilité et la portabilité” affirme Carmine Rimi, directeur de produit pour Kubernetes à Canonical.</p>
```
Comment (tbille) [inline templates/engage/fr/canonical-ajoute-containerd-a-ubuntu-kubernetes.html:29]:
```suggestion
      <p>“ Containerd est devenu le standard de l'industrie du container runtime. Il est concentré sur la simplicité, la stabilité et la portabilité” affirme Carmine Rimi, directeur de produit pour Kubernetes à Canonical.</p>
```

**Comment by tbille** [comment]
@steverydz Can we wait a bit before continuing this piece of work. I think there is still some work on the translations.

**Comment by SirSamTumless** [comment]
These are good to go

**Comment by steverydz** [comment]
@deadlight this is ready for re-review now

**Pull request merged**